ua_components/src/lib/Loading.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  // Block the rest of the page while 'loading' is non-empty
+
+  export let loading: string[];
+
+  let inner: HTMLDivElement;
+
+  $: if (loading.length > 0 && inner) {
+    inner.scrollTop = inner.scrollHeight;
```

Comment:
This doesn't really work well or look good yet (https://github.com/a-b-street/15m/pull/2 for context) but we can iterate on it